app/lib/selections/tools/pen.dart

```diff
@@ -21,6 +21,29 @@ class PenToolSelection extends ToolSelection<PenTool> {
                       zoomDependent: value ?? selected.first.zoomDependent))
                   .toList())),
       const SizedBox(height: 16),
+      // ! new checkBox
```

Comment:
I think this comment isn't necessary here

Reply:
done